api/proto/teleport/legacy/types/types.proto

```diff
@@ -6758,6 +6760,19 @@ message PluginSpecV1 {
   string generation = 11;
 }
 
+message PluginGithubSettings {
+  option (gogoproto.equal) = true;
+
+  // APIEndpoint is the address of Github API.
```

Comment:
this is only important if using github enterprise.  It's worth adding to the doc